frontend-react/src/App.js

```diff
@@ -0,0 +1,70 @@
+
+import {useState} from 'react'
+import {Switch, Route, Redirect} from 'react-router-dom'
+
+import Header from './Header.js';
+import Main from './pages/Main.js';
+import About from './pages/About.js';
+import Settings from './pages/Settings.js';
+
+import './App.css';
+
+const baseUrl = "http://explorehsk.com:5000/"
+
+async function getRandomWord() {
+    try {
+        const response = await fetch(baseUrl + `random`);
+        const jsonData = await response.json();
+        return jsonData;
+
+    } catch(err) {
+        console.log(err)
+    }
+}
+
+
+function App() {
+
+    const [randomWord, setRandomWord] = useState("什么")
+    const [hskLevel, setHskLevel] = useState(4)  // TODO: get this to be set with cookie
+    const [usePinyinAccents, setUsePinyinAccents] = useState(false)  // TODO: get this to be set with cookie
+
```

Comment:
You can use a custom hook to set it in brower's local storage with this code snippet : 
https://usehooks.com/useLocalStorage/

Reply:
This is very useful. Addressed in 3496208